docs/modules/operation/pages/provisioning/scalability.adoc

```diff
@@ -1,42 +1,42 @@
+The scalability of a management system is defined by its capacity for managing large numbers of entities coupled with its efficiency of managing the entities.
 
-Today, It is not uncommon for {page-component-title} deployments to find node entities with tens of thousands of physical interfaces being reported by SNMP agents due to virtualization (virtual hosts, interfaces, as well as networks).
-An NMS must be capable of using the full capacity every resource of its computing platform (hardware and OS) as effectively as possible in order to manage these environments.
-The days of writing scripts or single threaded applications will just no longer be able to do the work required an NMS when dealing with the scalability challenges facing systems and systems administrators working in this domain.
+Today, it is not uncommon for {page-component-title} deployments to find node entities with tens of thousands of physical interfaces being reported by SNMP agents due to virtualization (virtual hosts, interfaces, and/or networks).
+A monitoring system must be capable of using the full capacity every resource of its computing platform as effectively as possible.
+The days of writing scripts or single threaded applications no longer is sufficient to do the work required by a monitoring system at the scale of most networks today.
 
-== Parallelization and Non-Blocking I/O
+== Parallelization and non-blocking I/O
 
-Squeezing out every ounce of power from a management system’s platform (hardware and OS) is absolutely required to complete all the work of a fully functional NMS such as {page-component-title}.
-Fortunately, the hardware and CPU architecture of a modern computing platform provides multiple CPUs with multiple cores having instruction sets that include support for atomic operations.
-While these very powerful resources are being provided by commodity systems, it makes the complexity of developing applications to use them vs. not using them, orders of magnitude more complex.
-However, because of scalability demands of our complex IT environments, multi-threaded NMS applications are now essential and this has fully exposed the complex issues of concurrency in software development.
+Squeezing out every ounce of power from a management system’s platform is absolutely required to complete all the work of a fully functional monitoring system such as {page-component-title}.
+Fortunately, the hardware architecture of a modern computing platforms provides multiple CPUs with multiple cores having instruction sets that include support for atomic operations.
+Because of scalability demands of complex IT environments, multi-threaded monitoring solutions are now essential and this has fully exposed the complex issues of concurrency in software development.
```

Comment:
```suggestion
Because of scalability demands of complex IT environments, multi-threaded monitoring solutions are now essential. 
This has fully exposed the complex issues of concurrency in software development.
```

Reply:
```suggestion
Because of scalability demands of complex IT environments, multi-threaded monitoring solutions are now essential.
```